src/Contexts/AuthDialogContext/useAuthDialog.jsx

```diff
@@ -0,0 +1,13 @@
+import React, { useState } from 'react';
+
+export const useAuthDialog = () => {
+  const [authType,setAuthType] = useState(null);
+  const [creds,setCreds] = useState({});
+
+  const handleAuthCreds = (e) => {
```

Comment:
Suggestion: 
Since the function is empty, you can remove it.  
The creds state is also of not used anywhere, it can also be removed.

Reply:
fixed